apps/api/v2/src/ee/event-types/event-types_2024_06_14/inputs/create-phone-call.input.ts

```diff
@@ -0,0 +1,71 @@
+import { ApiProperty as DocsProperty } from "@nestjs/swagger";
+import { Transform } from "class-transformer";
+import { IsString, IsBoolean, IsOptional, IsEnum, Matches } from "class-validator";
+
+export enum TemplateType {
+  CHECK_IN_APPOINTMENT = "CHECK_IN_APPOINTMENT",
+  CUSTOM_TEMPLATE = "CUSTOM_TEMPLATE",
+}
+
+export class CreatePhoneCallInput {
+  @IsString()
+  @Matches(/^\+[1-9]\d{1,14}$/, {
+    message:
+      "Invalid phone number format. Expected format: +<CountryCode><PhoneNumber> with no spaces or separators.",
+  })
+  @DocsProperty({ description: "Your phone number" })
+  yourPhoneNumber!: string;
+
+  @IsString()
+  @Matches(/^\+[1-9]\d{1,14}$/, {
+    message:
+      "Invalid phone number format. Expected format: +<CountryCode><PhoneNumber> with no spaces or separators.",
+  })
+  @DocsProperty({ description: "Number to call" })
+  numberToCall!: string;
+
+  @IsString()
+  @DocsProperty({ description: "CAL API Key" })
+  calApiKey!: string;
+
+  @IsBoolean()
+  @DocsProperty({ description: "Enabled status", default: true })
+  enabled = true;
```

Comment:
To make typescript happy need to add `!` aka `enabled!`

Reply:
I don't see any ts error on this

Reply:
since the type is 'true' it should be fine without the ! indeed, ! would be needed in the type was boolean